wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -117,7 +119,9 @@
                         url += `?position=${(index)}`;
                         let CSRFToken = $('input[name="csrfmiddlewaretoken"]', orderform).val();
                         $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
-                            addMessage('success', `"${$(currentlySelected).data('page-title')}" has been moved successfully from ${currentPosition +1} to ${index +1}.`)
+                            const text = `"${$(currentlySelected).data('page-title')}" has been moved successfully from ${currentPosition +1} to ${index +1}.`
+                            const event = new CustomEvent('w-message:add',{ detail:{status:'stauts', text, clear:true, } });
+                            document.dispatchEvent(event);
```

Comment:
```suggestion
                            const text = `"${$(currentlySelected).data('page-title')}" has been moved successfully from ${currentPosition + 1} to ${index + 1}.`;
                            const event = new CustomEvent('w-message:add', { detail: { status: 'stauts', text, clear: true } });
                            document.dispatchEvent(event);
```

Just some formatting refinements.